postman/README.md

```diff
@@ -0,0 +1,15 @@
+#Authenticating with Postman
+
+Postman is a great way to quickly test all aspects of the API - from authentication to response bodies,
+to accepted params, etc... Postman supports the Onshape OAuth2.0 authentication out of the box. 
+With a little tweaking, Postman can also support API key authentication.
+
+## API Key Authentication
+ 
+How to setup Postman to correctly sign API requests:
+1. File->Import-> select this file: [Onshape Postman Collection.](./onshape.postman_collection.json).
+1. Now make and keep handy your secret key and access key from the [Onshape Dev Portal](https://dev-portal.onshape.com/).
+1. Add the environment variables `accessKey` and `secretKey` to the collection as seen here:
+![open environment variables in collection](./postman_open_env_vars.jpg)
```

Comment:
I'll provide a higher quality picture here (not sure if I can push it to the branch - I'll try).

Reply:
I think I gave you write access. Should work!